qxf2/qxf2-lambdas
# Test unique tech lambda on localstack

Test unique tech lambda on localstack:
* Deploy Lambda along with its dependencies
* Run test and delete lambda

Setup:
* cd to tests directory
* Start LocalStack using following command:
   `docker-compose up -d`
* Check LocalStack status:
   `localstack status`

Run Test:

* cd to unique_tech_notifier/test directory
* pytest -s -v test_unique_tech_notifier_lambda_localstack.py

Running Test:
![image](https://github.com/qxf2/qxf2-lambdas/assets/110374029/a288dfe8-373e-4da6-b1b3-870f4e712e5f)
